kalite/topic_tools/settings.py

```diff
@@ -12,16 +12,22 @@
 import os
 from django.conf import settings
 
+CHANNEL = getattr(settings, "CHANNEL", "khan")
```

Comment:
If the user doesn't specify CHANNEL in their settings module, could the following situation be problematic?

```python
from django.conf import settings
from kalite.topic_tools import settings as tt_settings

settings.CHANNEL  # AttributeError
tt_settings.CHANNEL  # "khan"
```